src/awkward/contents/recordarray.py

```diff
@@ -192,6 +181,16 @@ def simplified(
     ):
         return cls(contents, fields, length, parameters=parameters, backend=backend)
 
+    @property
+    def is_tuple(self):
+        return self._fields is None
+
+    @property
+    def as_tuple(self):
+        return RecordArray(
+            self._contents, None, self._length, parameters=None, backend=self._backend
```

Comment:
We need to drop the parameters because the type has changed. Only operations that return the same type can keep the parameters.

In fact, when turning a record into a tuple, overloads would almost certainly break. Most overloads are defined in terms of record fields, which are now gone.